www/src/public/js/controllers/builder/main.js

```diff
@@ -178,11 +212,14 @@
                             case 2:
                             case 3:
                                 var newList = createListFromStringV2(listStrs[i], listVersion);
-                                break;                            
+                                break;    
+                            case 4: //should we change how this works to use the new built in version?
```

Comment:
This entire switch/case statement should probably just be turned into an if/else.

```
if (listVersion == "cl" || listVersion == 1) {
  var newList = createListFromString(listStrs[i]);
} else {
  var newList = createListFromStringV2(listStrs[i], listVersion);
}
```